configs/platforms/windows-2012r2-x64.rb

```diff
@@ -9,10 +9,12 @@
   plat.add_build_repository "http://nexus.delivery.puppetlabs.net/service/local/nuget/nuget-pl-build-tools/"
 
   # We don't want to install any packages from the chocolatey repo by accident
+  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe update -y chocolatey"
```

Comment:
I *think* one alternative here is to use the newest choco and install `7zip.commandline` from our choco feed at http://nexus.delivery.puppetlabs.net/service/local/repositories/temp-build-tools/content/7zip.commandline/9.20.0.20150210/7zip.commandline-9.20.0.20150210.nupkg

That will get `7za` back into PATH, which is I believe what broke the builds.